scene/gui/label.cpp

```diff
@@ -431,7 +436,6 @@ void Label::_notification(int p_what) {
 			int outline_size = has_settings ? settings->get_outline_size() : theme_cache.font_outline_size;
 			int shadow_outline_size = has_settings ? settings->get_shadow_size() : theme_cache.font_shadow_outline_size;
 			bool rtl = (TS->shaped_text_get_inferred_direction(text_rid) == TextServer::DIRECTION_RTL);
-			bool rtl_layout = is_layout_rtl();
```

Comment:
What's the purpose of removing this and `size`, these values can't change, so there's no reason to call functions multiple times.

Reply:
I thought it's the same since they are just getters. if you want to revert this change I will do it.